README.md

```diff
@@ -80,6 +80,24 @@ shpwrite.download({
 Given a [GeoJSON](http://geojson.org/) FeatureCollection as an object,
 converts convertible features into Shapefiles and triggers a download.
 
+### `downloadMany(geojsonArray)`
```

Comment:
add `options` parameter as it is required. yes, i know it's not elsewhere either but...